### PR TITLE
Update puppeteer tp 2.1.1

### PR DESCRIPTION
**Pull request by pmerwin**
## Proposed changes

[//]: # (Update to 2.1.1 (latest version) of puppeteer

## Types of changes
Maintenance
[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/devtools/package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "@wdio/protocols": "6.0.0-alpha.1",
     "@wdio/utils": "6.0.0-alpha.1",
     "chrome-launcher": "^0.11.1",
-    "puppeteer-core": "^1.18.1"
+    "puppeteer-core": "^2.1.1",
+    "puppeteer-firefox": "^0.5.0"
```
Comment (christian-bromann) [inline packages/devtools/package.json]: We have this already in `optionalDependencies` so this should be removed.
```suggestion
```

Comment (pmerwin) [reply packages/devtools/package.json]: OK I will need to fix the unit tests for that , too, let me see what I can do.

**Comment by christian-bromann** [comment]
ping @pmerwin

**Comment by christian-bromann** [comment]
Closing due to inactivity.

@pmerwin if you still facing this issue please rebase your branch and make a new PR. Thanks!

**Pull request closed without merging**